### ISSUE TITLE
Suggestion - option to remove quotes (single and/or double) from YAML

### ISSUE THREAD

**Issue by S-Wil** [kometenstaub/csv-to-md#8]
In my files I have quotes (they happen to be double) around text in order to preserve commas.  These are removed in the body but remain the YAML.  It would be great to have an option to remove surrounding double (or single) quotes from YAML.

Thanks

**Comment by kometenstaub** [comment]
Could you post an example?

Quotes are currently stripped but [added back](https://github.com/kometenstaub/csv-to-md/blob/6de08f951b8e75357669f14c4b16234b064ac6eb/csv_to_md-file.py#L284) to make sure that they are not separate YAML values.

The option to specify that there are multiple values in a column separated by e.g. a comma is made to make it possible to have multiple values. 

The quotes are added to make sure that something which is supposed to be one value is not separated by commas which could be in the text.

Do you think that there shouldn't be any quotes in general around YAML values?

**Comment by S-Wil** [comment]
Yes I understand why you are adding them back now.  In my case (which is a slightly different use case) I have any mult-value YAML already together in a single column in the CSV so it's not really an issue for me and the quotes don't add any value but make it messy in the edit view.

I prefer to only have tags and aliases in the YAML (or anything that Obsidian can use), then additional metadata in the body with "column name:: data" for dataview work, then some free form fields without column headings.  For example:
/---
Aliases: [Matthew Smith, Matt Smith]
Tags: [ABC_Co]
/---
Full Name:: Matthew Smith
Title:: Manager
Location:: London
Free form notes about Matthew here...
